src/main/java/org/folio/inventory/resources/Items.java

```diff
@@ -314,7 +326,7 @@ private Item requestToItem(JsonObject itemRequest) {
             .withPermanentLocationId(permanentLocationId)
             .withTemporaryLocationId(temporaryLocationId)
             .withTemporaryLoanTypeId(temporaryLoanTypeId)
-            .withCopyNumbers(copyNumbers)
+            .withCopyNumber(itemRequest.getString("copyNumber"))
```

Comment:
Folks seem to likely constants for the names of the JSON properties. Given this is used in multiple places, I think it might be Worthing following suite with this